src/components/cms/SkillCardScrollList/SkillCard.js

```diff
@@ -74,12 +74,31 @@ class SkillCard extends Component {
   };
 
   componentDidMount = () => {
+    window.addEventListener('resize', () => {
+      this.updateWindowDimensions();
+      this.updateBtnsDisplay();
+    });
     this.updateWindowDimensions();
   };
 
   updateWindowDimensions = () => {
     let scrollCards = 1;
     switch (true) {
+      case window.innerWidth >= 3400:
+        scrollCards = 10;
+        break;
+      case window.innerWidth >= 3100:
+        scrollCards = 9;
+        break;
+      case window.innerWidth >= 2700:
+        scrollCards = 8;
+        break;
+      case window.innerWidth >= 2400:
+        scrollCards = 7;
+        break;
+      case window.innerWidth >= 2100:
+        scrollCards = 6;
```

Comment:
Try to make these `window.innerWidth` conditions consistent with the previous ones. As the skillcard size is around 180px.

Reply:
Sir, the skillcard size is 280px. We cannot add 280px straight to 840 because i have used relative width for sidebar. When I am adding 280px, the right arrow is getting misaligned with the cards.
I have given width = 15% for the sidebar.
So, I am using the formula `window.innerWidth * 0.15 + 10px[Margin to Sidebar] + 280px[Box size]*n[number of boxes to display]` formula to get the required width for switch statements.
As this formula calculates it value from n, therefore i replaced switch with a for loop.


Now I am gettings desired result in the webpage